pkgs/servers/http/pomerium/common.nix

```diff
@@ -0,0 +1,27 @@
+{ fetchFromGitHub
+, lib
+}:
+let
+  version = "0.19.0";
+  srcSha256 = "sha256:0s5ji1iywymzxlv89y3ivl5vngkifhbpidpwxdrh969l3c5r4klf";
+  vendorSha256 = "sha256:1p78nb7bryvs7p5iq6ihylflyjia60x4hd9c62ffwz37dwqlbi33";
+  yarnSha256 = "sha256:1n6swanrds9hbd4yyfjzpnfhsb8fzj1pwvvcg3w7b1cgnihclrmv";
+in
+{
```

Comment:
Instead of using this extra common file I would suggest to use the attributes from pomerium for the cli and add missing ones to passthru.

Reply:
This isn't for the CLI; it's for the UI which is a dependency of pomerium. OTOH, putting this directly on the UI would be a bit... weird.

I'll leave this as is for the moment but yeah, it's probably worth refactoring the UI derivation back into the main package.

Reply:
@lukegb agree it's a little clunky -- I split the UI derivation out in an attempt to resolve the issue that adding package.json ended up resolving. Happy to recombine them before this is merged if that'd help.